.github/CODEOWNERS

```diff
@@ -13,6 +13,3 @@
 /src/cli/** @lurk-lab/lurk-cli-tools
 /fcomm/** @lurk-lab/lurk-cli-tools
 /clutch/** @lurk-lab/lurk-cli-tools
-
-# LEM
-/src/lem/** @lurk-lab/lurk-lem
```

Comment:
We have soundness-sensitive (circuit) code in the `lem` folder. How should we require specialized review on those files?